src/Forvaltningsportalen/KartVelger.js

```diff
@@ -77,7 +77,7 @@ const KartVelger = props => {
       </div>
       <div className="map_ref">
         {tilgjengelige_data[props.aktivtFormat].navn} |{" "}
-        <a href={tilgjengelige_data[props.aktivtFormat].url}>
+        <a tabIndex="-1" href={tilgjengelige_data[props.aktivtFormat].url}>
```

Comment:
Vil ikke denne bli utilgjengelig for en person uten keyboard nå?

Reply:
Ja, det blir utilgjengelig til en bruker uten mus som kan bare bruke keyboard. Kan ta det bort så det blir tilgjengelig igjen. Men kanskje linken bør åpnes i et nytt vindu? Akkurat nå det åpnes i selve vindu, som er litt uprakstisk synes jeg.